src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public Colors getRandomColor() {
```

Comment:
Return color in String type

Reply:
Why? Isn't it better to return them as Colors type to avoid unnecessary conversion later? Thanx